src/main/java/com/salesforce/storm/spout/sideline/recipes/trigger/zookeeper/ZookeeperWatchTrigger.java

```diff
@@ -327,6 +325,14 @@ private SidelineRequestIdentifier generateSidelineRequestIdentifier(
         return new SidelineRequestIdentifier(identifier.toString());
     }
 
+    private String generateIdFromJson(final String dataJson) throws NoSuchAlgorithmException {
```

Comment:
Functionally the same as what was there before, just no longer dependent upon javax.